persistence/pg.go

```diff
@@ -131,6 +148,23 @@ func (p *PostgresPersister) Get(ctx context.Context, hash lntypes.Hash) (*Invoic
 	return invoice, htlcs, nil
 }
 
+func (p *PostgresPersister) GetSettledInvoices(ctx context.Context, numMaxInvoices, invoiceOffset int) ([]*dbInvoice, error) {
+	var dbInvoices []*dbInvoice
+	err := p.conn.RunInTransaction(ctx, func(tx *pg.Tx) error {
+		return tx.ModelContext(ctx, &dbInvoices).
+			Where("sequence_num>?", invoiceOffset).
+			Where("settled = true").
```

Comment:
Don't we want to allow querying invoices for which settlement has been requested? This is different from invoice that are only accepted.

Reply:
Will add invoices for which settlement has been requested